Dockerfile

```diff
@@ -29,6 +29,10 @@ RUN git clone https://github.com/akesterson/cmdarg.git /tmp/cmdarg \
 RUN git clone https://github.com/akesterson/shunit.git /tmp/shunit \
     && cd /tmp/shunit && make install && rm -rf /tmp/shunit
 
+# Go tool for internationalization and localization
+RUN go get github.com/JiangtianLi/gettext/... \
```

Comment:
Just echoing others and calling out we can't include this change as is, the repo has to be publically accessible.